hbase-shaded/pom.xml

```diff
@@ -451,6 +451,10 @@
                     <pattern>net/</pattern>
                     <shadedPattern>${shaded.prefix}.net.</shadedPattern>
                   </relocation>
+                  <relocation>
+                    <pattern>org.agrona</pattern>
```

Comment:
What is the reason to shade this dependency?

I do not mean you should not do this, just want to know th reason...

Reply:
This will shade and pack the dependency in release. according to @busbey It seems working.

Reply:
This is not an 'answer' but anyway, we have shaded almost all the known dependencies here so I think it is fine to also shade this one.